src/entities/course/types.ts

```diff
@@ -15,3 +15,10 @@ export type Course = {
     accentColor: string;
   };
 };
+
+export type CourseCardProps = Pick<
+  Course,
+  'title' | 'iconSrc' | 'startDate' | 'detailsUrl' | 'mode' | 'language' | 'backgroundStyle'
+>;
```

Comment:
We often talking about code style consistency in our codebase. In all our components we define the props type inside the component file itself. Probably another topic that we need to think and note in our documentation

Reply:
Done